ta/pkcs11/src/serializer.c

```diff
@@ -112,3 +172,27 @@ enum pkcs11_rc serialargs_get_session_from_handle(struct serialargs *args,
 
 	return PKCS11_CKR_OK;
 }
+
+/*
+ * serialize - serialize input data in buffer
+ *
+ * Serialize data in provided buffer.
+ * Insure 64byte alignment of appended data in the buffer.
+ */
+enum pkcs11_rc serialize(char **bstart, size_t *blen, void *data, size_t len)
+{
+	char *buf = NULL;
+	size_t nlen = *blen + len;
```

Comment:
add an overflow check here to avoid shrinking `buf` when `len` is huge

Reply:
As this is called from a secure world based on hopefully reliable content, if think the proposed added check is valuable and should even panic the TA.

Reply:
and I don't see a check for the 64byte-alignment

Reply:
> and I don't see a check for the 64byte-alignment

Alignment of what?

Reply:
Aha, I see the comment above. @etienne-lms, should we just drop that comment?

Reply:
Note that this 64-byte alignment is also referenced by the caller of this function, so you should investigate whether some (implicit) contract between functions is violated here.